### ISSUE TITLE
Building @nativescript/core for Angular requires nativescript-dev-webpack with version at least 1.3.0

### ISSUE THREAD

**Issue by zbranzov** [NativeScript/NativeScript#8064]
**Environment**
Provide version numbers for the following components (information can be retrieved by running `tns info` in your project folder or by inspecting the `package.json` of the project):
 - CLI: 6.2
 - Cross-platform modules:6.2
 - Android Runtime:6.2
 - iOS Runtime:6.2

**Describe the bug**
If an angular project has dependency to "nativescript-dev-webpack": "^1.0.0", the build will fail because of a wrong version check throwing : **Building @nativescript/core for Angular requires nativescript-dev-webpack with version at least 1.3.0. Please upgrade: npm i nativescript-dev-webpack --save-dev.**

**To Reproduce**
Add "nativescript-dev-webpack": "^1.0.0" to angular project

**Expected behavior**
App is successfully built.

**Additional context**
Platform ios successfully added. v6.2.0
Preparing project...
Hash: 5ba88bf73a348f5372e3
Version: webpack 4.27.1
Time: 39572ms
Built at: 11/06/2019 3:04:52 PM
                                            Asset       Size                                          Chunks             Chunk Names
                                             0.js   58.1 KiB                                               0  [emitted]  
                                        bundle.js   18.2 KiB                                          bundle  [emitted]  bundle
                                     package.json  159 bytes                                                  [emitted]  
                                       runtime.js   12.9 KiB                                         runtime  [emitted]  runtime
tns_modules/tns-core-modules/inspector_modules.js  626 bytes  tns_modules/tns-core-modules/inspector_modules  [emitted]  tns_modules/tns-core-modules/inspector_modules
                                        vendor.js   9.84 MiB                                          vendor  [emitted]  vendor
Entrypoint bundle = runtime.js vendor.js bundle.js
Entrypoint tns_modules/tns-core-modules/inspector_modules = runtime.js vendor.js tns_modules/tns-core-modules/inspector_modules.js
[../$$_lazy_route_resource lazy recursive] ../$$_lazy_route_resource lazy namespace object 160 bytes {bundle} [built]
[../../src/geolocation.ts] /Users/branzov/Git/nativescript-geolocation/src/geolocation.ts 15 KiB {0} [built]
[./app.css] 1.43 KiB {bundle} [built]
[./app/app-routing.module.ts] 304 bytes {bundle} [built]
[./app/app.component.ngfactory.js] 1.67 KiB {bundle} [built]
[./app/app.component.ts] 140 bytes {bundle} [built]
[./app/app.module.ngfactory.js] 6.86 KiB {bundle} [built]
[./app/app.module.ts] 128 bytes {bundle} [built]
[./app/home/home-routing.module.ts] 271 bytes {0} [built]
[./app/home/home.component.ngfactory.js] 4.66 KiB {0} [built]
[./app/home/home.component.ts] 2.54 KiB {0} [built]
[./app/home/home.module.ngfactory.js] 2.65 KiB {0} [built]
[./app/home/home.module.ts] 132 bytes {0} [built]
[./main.ts] 1.16 KiB {bundle} [built]
[./package.json] 138 bytes {bundle} {tns_modules/tns-core-modules/inspector_modules} [optional] [built]
    + 523 hidden modules
Webpack compilation complete.
**Building @nativescript/core for Angular requires nativescript-dev-webpack with version at least 1.3.0. Please upgrade: npm i nativescript-dev-webpack --save-dev.**

**Comment by nuochong** [comment]
me to

Error
```
Webpack build done!
Building @nativescript/core for iOS requires NativeScript CLI with version at least 6.2.0. Please upgrade your NativeScript CLI version (npm i -g nativescript).
```

Two months ago, everything was normal. Today, I reinstalled the dependency package, which resulted in this situation. I had to force the upgrade, but after the upgrade, vuex was not compatible and IOS ran with an error. It can't be downgraded. Native script is the second major version change in a year. Each time, it's a lot of errors. It feels like it's always dealing with the errors of the framework itself. And the last change made nearly 90% of the NS plug-ins unusable. Good helpless, heart tired!
